MyMovies/View Controllers/MyMoviesTableViewController.swift

```diff
@@ -7,89 +7,114 @@
 //
 
 import UIKit
+import CoreData
 
 class MyMoviesTableViewController: UITableViewController {
-
+    
+    let movieController = MovieController()
+    
+    lazy var fetchedResultsController: NSFetchedResultsController<Movie> = {
+        let fetchRequest: NSFetchRequest<Movie> = Movie.fetchRequest()
+        fetchRequest.sortDescriptors = [NSSortDescriptor(key: "hasWatched", ascending: true)]
+        let context = CoreDataStack.shared.mainContext
+        let frc = NSFetchedResultsController(fetchRequest: fetchRequest, managedObjectContext: context, sectionNameKeyPath: "hasWatched", cacheName: nil)
+        frc.delegate = self
+        try! frc.performFetch()
```

Comment:
Totally fine to use this bang to test things out and make sure the code works especially when you are in a time constraint, but make sure you leave a //FIXME so you can write it out properly in a Do Try Catch block.